src/main/java/com/internhub/data/Main.java

```diff
@@ -114,15 +103,17 @@ private static void scrapePositions(String name) {
     }
 
     private static void scrapePositions(List<Company> companies) {
```

Comment:
make it clearer that this method is for single threading. Why are we still maintaining single threading as well?